### PR TITLE
Limit Global Styles: Handle launchbar action in Shadow DOM

### PR DESCRIPTION
**Pull request by mmtr**
Part of https://github.com/Automattic/dotcom-forge/issues/1013

#### Proposed Changes

The launchbar for Atomic sites is displayed in a Shadow DOM which doesn't have access to the styles and scripts enqueued by `wpcom_global_styles_enqueue_assets`, so this PR adds a workaround that injects the needed styles and scripts within the launchbar action.

#### Testing Instructions

See https://github.com/Automattic/wpcomsh/pull/1192

### CODE REVIEW

**Comment by BogdanUngureanu** [comment]
e2e tests are failing. Maybe it needs a rebase? :) @mmtr

**Comment by BogdanUngureanu** [comment]
Nice work, Miguel! I've tested it on my AT environment but it looks like there are some issues on mobile resolutions :( 
Screenshot 2022-12-14 at 13 55 21

**Comment by mmtr** [comment]
@BogdanUngureanu Looks like `wpcomsh` was missing some styles that were added to WPCOM. I updated https://github.com/Automattic/wpcomsh/pull/1192 with those so I think your issues should be solved now:

Dec-14-2022 13-41-47

**Review by BogdanUngureanu — APPROVED** [review]
LGTM! 🚢 🥇

File: apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/wpcom-global-styles-view.js [hunk 1]
```diff
@@ -9,11 +9,22 @@ function recordEvent( button ) {
 	} );
 }
 
-( function () {
-	const popoverToggle = document.querySelector( '.launch-bar-global-styles-toggle' );
-	const popover = document.querySelector( '.launch-bar-global-styles-popover' );
-	const upgradeButton = document.querySelector( '.launch-bar-global-styles-upgrade-button' );
-	const previewButton = document.querySelector( '.launch-bar-global-styles-preview-link' );
+document.addEventListener( 'DOMContentLoaded', () => {
```
Comment (simison) [inline apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/wpcom-global-styles-view.js:12]: Really minor drive-by comment but there's a [`wp.dom-ready`](https://github.com/WordPress/gutenberg/tree/3245d1e0518d6a2fa565266981fe0fab8a984a8e/packages/dom-ready#readme) helper which is a bit more thorough than standard `DOMContentLoaded` listener.
